action.yml

```diff
@@ -1,8 +1,25 @@
-name: 'ggshield Local Action'
-description: 'Scan commits for security incidents (using repository image)'
+name: 'ggshield scan'
+description: 'Scan commits for hardcoded secrets and security policy breaks.'
+author: GitGuardian <support@gitguardian.com>
+
 branding:
   icon: 'shield'
   color: 'blue'
+  
+inputs:
+  args:
+    description: |
+      Arguments to be passed to ggshield secret scan
+      Options:        
+        --json                        Output results in JSON format  [default: False]
+        --show-secrets                Show secrets in plaintext instead of hiding them.
+        --all-policies                Present fails of all policies (Filenames, FileExtensions,
+                                      Secret Detection). By default, only Secret Detection is shown.
+        --exit-zero                   Always return a 0 (non-error) status code, even if incidents are found.
+        -v, --verbose                 Verbose display mode.
```

Comment:
Since `runs.args` already contains `-v`, the output is always going to be verbose. This means listing `--verbose` in the arguments is confusing because it won't affect the output.

Reply:
Agreed, I realized this after I created the PR.